tools/EMLassemblyline/make_eml.xml

```diff
@@ -0,0 +1,67 @@
+<tool id="makeeml" name="Make EML" version="0.1.0+galaxy0">
+    <description> Create EML from EAL templates</description>
+    <requirements>
+        <requirement type="package" version="4.3.1">r-base</requirement>
+        <requirement type="package" version="3.5.5">r-emlassemblyline</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    mkdir output_template &&
+    cd output_template &&
+    #for $input in $templates
+        cp '$input' '${input.element_identifier}.txt' &&
+    #end for
+    find . -type f -empty -print -delete &&
+    Rscript '$__tool_directory__/make_eml.R'
+    ]]></command>
+    <inputs>
+        <param name="templates" type="data_collection" type_collection="list" format="tabular" label="Upload all EAL templates" help= "Input a data collection." />

```

Comment:
Why are data collection needed. Sorry I missed the discussion.

Usually multiple="true" is enough?

Reply:
Yes, multiple = true can be ok, but here, the idea is to create several tools who can deal with these "metadata templating files" who are supposed to be in a same folder. Using data collection is a way to have a similar manner to work on these templating files gathered in the same "place"